### PR TITLE
Add vocab definitions for secretKeyJwk and secretKeyMultibase

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
- Addresses issue https://github.com/w3c/vc-di-eddsa/issues/4
- Addresses @OR13's comment https://github.com/w3c/vc-data-integrity/pull/135/files#r1279821900

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Comment by OR13** [comment]
See also the comments on https://github.com/w3c/vc-data-integrity/pull/135

**Review by Wind4Greg — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

File: vocab/security/vocabulary.yml [hunk 1]
```diff
@@ -235,6 +235,19 @@ property:
       - label: ed25519-2020
         url: https://w3c-ccg.github.io/lds-ed25519-2020/
 
+  - id: secretKeyMultibase
+    label: Secret key multibase
+    domain: sec:Multikey
+    range: xsd:string
```
Comment (msporny) [inline vocab/security/vocabulary.yml:257]:
```suggestion
    range: sec:multibase
```

**Review by msporny — CHANGES_REQUESTED** [review]
Minor range fix, otherwise LGTM.

File: vocab/security/vocabulary.yml [hunk 2]
```diff
@@ -235,6 +235,19 @@ property:
       - label: ed25519-2020
         url: https://w3c-ccg.github.io/lds-ed25519-2020/
 
+  - id: secretKeyMultibase
+    label: Secret key multibase
+    domain: sec:Multikey
+    range: sec:multibase
```
Comment (iherman) [inline vocab/security/vocabulary.yml]: The `sec:multibase` class does not exist in the current vocabulary (unless I miss it in the maze of PR-s). Note that, for a class, it should be `sec:Multibase` (ie, capital 'M')

**Review by iherman — CHANGES_REQUESTED** [review]

File: vocab/security/vocabulary.yml (hunk 2 above)
Comment (OR13) [reply vocab/security/vocabulary.yml]: https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L245

^ related (and in the vocabulary already).

I agree regarding the capitalization

Comment (dlongley) [reply vocab/security/vocabulary.yml]: `sec:multibase` isn't a class, it's a datatype, like `xsd:string`. This needs the same consideration as `sec:cryptosuiteString`.

Comment (dmitrizagidulin) [reply vocab/security/vocabulary.yml]: @iherman @dlongley so, do I need to do anything in this PR, with respect to `sec:multibase` data type?

Comment (OR13) [reply vocab/security/vocabulary.yml]: > `sec:multibase` isn't a class, it's a datatype, like `xsd:string`. This needs the same consideration as `sec:cryptosuiteString`.

Then the capitalization makes sense as is, here is another reference to data types in the v2 context for comparison:

- https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L62

Comment (iherman) [reply vocab/security/vocabulary.yml]: > `sec:multibase` isn't a class, it's a datatype, like `xsd:string`. This needs the same consideration as `sec:cryptosuiteString`.

Ah! O.k., understood; the missing capitalization is fine then. However, just as for cryptosuite, the datatype must be properly defined in the spec and then the vocabulary yml file must be updated.

I would propose that this PR would include the datatype definitions in the spec and when all these types of PRs are merged I will take care of incorporating the datatype definitions into the vocabulary specification itself in a separate PR.

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, objections clarified until there were no objections remaining, merging.

**Pull request merged**